package.json

```diff
@@ -72,7 +71,9 @@
     "eslint-plugin-babel": "^3.3.0",
     "eslint-plugin-import": "^1.9.2",
     "eslint-plugin-mocha": "^3.0.0",
+    "fancy-log": "^1.3.0",
```

Comment:
I got rid of it. There was an error that happened earlier that said that the module was missing so I tried fixing it by adding it as a dev-dep. That was unneccessarry. I just did `rm -rf node_modules && npm install && gulp once` and it worked out fine.